Jint.Tests.Test262/Jint.Tests.Test262.csproj

```diff
@@ -1,6 +1,8 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
     <TargetFrameworks>net461;net5.0</TargetFrameworks>
+    <AssemblyOriginatorKeyFile>..\Jint\Jint.snk</AssemblyOriginatorKeyFile>
+    <SignAssembly>true</SignAssembly>
```

Comment:
Feels like I asked the same question recently, but what's the reason?

Reply:
Just to get access to internal methods, like Jint.Tests has